internal/daysteps/daysteps.go

```diff
@@ -1,20 +1,90 @@
 package daysteps
 
 import (
+	"errors"
+	"fmt"
+	"log"
+	"strconv"
+	"strings"
 	"time"
+
+	"github.com/Yandex-Practicum/tracker/internal/spentcalories"
 )
 
 const (
 	// Длина одного шага в метрах
-	stepLength = 0.65
+	stepLength float64 = 0.65
 	// Количество метров в одном километре
-	mInKm = 1000
+	mInKm float64 = 1000
 )
 
 func parsePackage(data string) (int, time.Duration, error) {
-	// TODO: реализовать функцию
+	// Разделить строку на слайс строк
+	parts := strings.Split(data, ",")
+
+	// Проверить, чтобы длина слайса была равна 2
+	if len(parts) != 2 {
+		log.Printf("неверный формат данных: %s", data)
+		return 0, 0, errors.New("неверный формат данных")
+	}
+
+	// Преобразовать первый элемент слайса (количество шагов) в тип int
+	steps, err := strconv.Atoi(parts[0])
+	if err != nil {
+		log.Printf("неверный формат количества шагов: %s", parts[0])
+		return 0, 0, errors.New("неверный формат количества шагов")
```

Comment:
1. Теряешь сообщение об ошибки
2. Ошибку лучше писать на английском
```suggestion
		return 0, 0, fmt.Errorf("conversion error: %w", err)
```